openfisca_web_api_preview/app.py

```diff
@@ -44,6 +44,10 @@ def get_variable(id):
             raise abort(404)
         return jsonify(variable)
 
+    @app.route('/swagger')
```

Comment:
What do you think about `/spec`, `/help` or `/manual`? `swagger` is a brand name, and it has since then be superseded by “OpenAPI”. This name still has the benefit of being the same as the current `/swagger` endpoint, but things have changed since then.